src/modules/bulletproofs/rangeproof_impl.h

```diff
@@ -209,31 +209,45 @@ static int secp256k1_bulletproof_rangeproof_verify_impl(const secp256k1_ecmult_c
         size_t j;
 
         /* Commit to all input data: min value, pedersen commit, asset generator, extra_commit */
```

Comment:
Is `age` below the asset generator? Just double-checking that the asset generator has not been forgotten.

Reply:
The asset generator is `value_gen` and it is actually committed, so this is OK.